### ISSUE TITLE
Details element cannot be interacted with using Voice Control

### ISSUE THREAD

**Issue by 36degrees** [alphagov/reported-bugs#54]
**Upstream bug:** https://bugs.webkit.org/show_bug.cgi?id=220086
**Date:** 2020-12-22
**Reported by:** Oliver Byford, on behalf of Craig Abbott (@abbott567)
**Related to:** https://github.com/alphagov/govuk-frontend/issues/2061

---

### Overview

Given the following markup:

```html
<a href="https://www.google.com">Google</a>

<br><br>

<details>
<summary>Details</summary>
<p>These are the details!</p>
</details>

<br><br>

<button type="button">Click me</button>

<br><br>

<a href="https://www.google.com">Google</a>
```

With Voice Control enabled, say 'Click Details'

Expected behaviour: The details element should be expanded.
Actual behaviour: Nothing happens.

Alternatively, you might try saying 'Show numbers' –

Expected behaviour: A number overlay should appear near the 'Details' summary, allowing you to interact with it by saying 'Click [number]'
Actual behaviour: No number overlay is shown for the 'Details' summary element (see screenshot)

As tested on macOS 10.15 / Safari 14
